plugins/rest/resturl/rest_url.go

```diff
@@ -66,4 +92,39 @@ const (
 const (
 	// Routes is rest static route path
 	Routes = "/vpp/dump/v1/routes"
+	// Arps is rest ARPs path
+	Arps = "/vpp/dump/v1/arps"
+	// ProxyArps is rest proxy ARPs path
+	ProxyArps = "/vpp/dump/v1/proxyarp"
+	// PArpIfs is rest proxy ARP interfaces path
+	PArpIfs = "/vpp/dump/v1/proxyarp/interfaces"
+	// PArpRngs is rest proxy ARP ranges path
+	PArpRngs = "/vpp/dump/v1/proxyarp/ranges"
+)
+
+// L4 plugin
+const (
+	// Sessions is rest L4 configuration path
+	Sessions = "/vpp/dump/v1/sessions"
+)
+
+// Command
+const (
+	// Command allows to put CLI command to the rest
+	Command = "/vpp/command"
```

Comment:
Seems like an overkill to define ALL of the URLs here. We should use README for documentation purposes.

Reply:
Unless we have a way how to list all available URLs, I'd keep it in the readme. Not everyone is going to use a browser, and showing an index list in the console is not very convenient.

Reply:
You might have misunderstood. I didn't advocate against keeping anything in README, quite the contrary actually. I was suggesting to keep full list of supported URLs in the README and not have separate package with the URLs. For now we could just keep the `vpp/dump/v1` stuff here. However after some recent discussions regarding the meta model, it might actually be moved next to the keys in models most likely.

Reply:
Also, the index list handled by `/`, can be served in format that is requested by client. If we are not requesting from browser, we can serve JSON.